genomepy/cli.py

```diff
@@ -87,8 +112,7 @@ def get_install_options():
     install_options = general_install_options
 
     for name in genomepy.provider.ProviderBase.list_providers():
-        p = genomepy.provider.ProviderBase.create(name)
-        p_dict = p.list_install_options()
+        p_dict = genomepy.provider.ProviderBase.list_install_options(name)
```

Comment:
@simonvh. I'm suddenly getting errors in the annotations branch from this change... No idea why that's only showing up now, but its fixed by undoing the change (actually it's still a one liner: `p_dict = genomepy.provider.ProviderBase.create(name).list_install_options()`). I'm not missing anything right?

PS can this branch be removed now it's been merged?